tiup/tiup-component-dm-prune.md

```diff
@@ -0,0 +1,25 @@
+---
+title: tiup dm prune
+---
+
+# tiup dm prune
+
+When [scaling in the cluster](/tiup/tiup-component-dm-scale-in.md), a small amount of meta-information in etcd is not cleaned up, which usually causes no problem. If you need to clean the meta-information up, you can manually execute the `tiup cluster prune` command.
```

Comment:
```suggestion
When you [scale in the cluster](/tiup/tiup-component-dm-scale-in.md), a small amount of metadata in etcd is not cleaned up, which usually causes no problem. If you need to clean up the metadata, you can manually execute the `tiup cluster prune` command.
```
Is this `metadata` or `meta-information`? @lucklove

Reply:
I think both is OK...